### ISSUE TITLE
Default value of stop_lying_iter

### ISSUE THREAD

**Issue by dkobak** [KlugerLab/FIt-SNE#44]
In http://jmlr.org/papers/volume15/vandermaaten14a/vandermaaten14a.pdf, early exaggeration 12 is used for the first 250 iterations. This is also the (unchangeable) default in https://github.com/lvdmaaten/bhtsne/blob/master/tsne.h#L45. However, FIt-SNE's default in all three wrappers is 200.

Is there any particular reason to use 200? Otherwise I'd set the default to 250, for historical consistency.

PS. When I was checking the parameter values now, I noticed that `mom_switch_iter`, `momentum`, `final_momentum`, and `learning_rate` are all hard-coded into the C++ code. Wouldn't it make sense to make them adjustable in wrappers? It's easy to do, but would change the format of the input files (shouldn't be a big deal?). Not that I need to change any of these four parameters :-)

**Comment by linqiaozhi** [comment]
You are right--this should be changed to 250.  I thought the default in the other implementations was 200, but it looks like that is not the case...I must have confused it with the `learning_rate` which is set to 200 by default.  Thanks for catching it.

Of the other parameters you listed, `learning_rate` could make the most difference for large N, I think. I have never touched the momentum related parameters, so I don't know whether those are important or not. I agree...making them all parameters would make the most sense though.

**Comment by dkobak** [comment]
Regarding the `learning_rate`, I noticed that in `scanpy` the default is set to 1000: https://scanpy.readthedocs.io/en/latest/api/scanpy.api.tl.tsne.html. I did not experiment it with it myself, but I guess the scanpy team did and found it helpful to change it. So yes, I'd definitely suggest to make `learning_rate` ann adjustable parameter. Regarding `mom_switch_iter`, `momentum`, and `final_momentum`, I have also no idea but if we get around making learning rate adjustable, we can as well make all of them adjustable at the same time.
